pom.xml

```diff
@@ -102,6 +102,8 @@
     <spark.version>1.6.1</spark.version>
     <scala.version>2.10.4</scala.version>
     <scala.binary.version>2.10</scala.binary.version>
+    <springboot.version>1.3.6.RELEASE</springboot.version>
```

Comment:
Why, 1.3.6? It looks like there is a 1.3.7 and the "current" is tagged as "1.4.0". Is there a reason that the older 1.3.6 is being used?